htdocs/core/modules/stock/doc/pdf_standard.modules.php

```diff
@@ -289,8 +289,8 @@ public function write_file($object, $outputlangs, $srctemplatepath = '', $hidede
 				$pdf->MultiCell(0, 3, ''); // Set interline to 3
 				$pdf->SetTextColor(0, 0, 0);
 
-				$tab_top = 80 + $top_shift;
-				$tab_top_newpage = (empty($conf->global->MAIN_PDF_DONOTREPEAT_HEAD) ? 42 + $top_shift : 10);
+				$tab_top = 65 + $top_shift;
```

Comment:
This file change seems not related to the PR that fix import with the != 'separate' . Can you repush without this file modifier ?

Reply:
Hi @eldy , i've already undone the modification.